### ISSUE TITLE
Zoom level not persisting on Zorin OS

### ISSUE THREAD

**Issue by svalbardsleeperdistrict** [standardnotes/forum#1561]
When I click zoom in view settings, the zoom gets reset on the next startup of the app.

**To Reproduce**
1. Launch the app on Zorin OS 16.1 64-bit
2. Click on View->Zoom In
3. Close the app, then relaunch it
4. Check if the zoom level you had selected has persisted

**Expected behavior**
Zoom level being automatically set to the previous level on startup

**Desktop:**
 - OS: Zorin OS 16.1 64-bit
 - Browser: N/A
 - Version 3.13.0

**Comment by adrihut** [comment]
Hi @svalbardsleeperdistrict!

I'm on Fedora, zoom level persists for me; this tells me the behavior is related to your environment. I'll spin-up a copy of Zorin on my end to try and replicate it.


To make sure I install the right stuff...
- What installation method are you using with your copy of SN—**snap, AppImage, ...**?
- What Zorin edition are you using—**Core or Lite**?
- What windowing system are using—**X11 or Wayland**?
- Who's the manufacturer of your system's graphics component—**AMD, Intel, or Nvidia**?


Also, what troubleshooting have you attempted thus far?

**Comment by svalbardsleeperdistrict** [comment]
Hi, thank you for the reply!

- I'm using the flatpak from dl.flathub.org
- Using Zorin Core edition
- X11 windowing system
- System graphics by Nvidia

Even though I've tried tons of distros over the years, I'm still kind of inexperienced at more technical stuff with Linux, so I haven't been able to attempt any troubleshooting so far. I will also check if the issue happens in Windows 10  which I sometimes use in a dualbooting installation.

Edit: I just tried it again on Zorin, and the zoom level was kept after I restarted the app. The issue may only be happening between system reboots/bootups - will check this too along with the above.
